### ISSUE TITLE
Connect-pnponline -url -interactive doesnot work

### ISSUE THREAD

**Issue by chandan805** [SharePoint/sp-dev-docs#9924]
### Target SharePoint environment

SharePoint Online

### What SharePoint development model, framework, SDK or API is this about?

not applicable

### Developer environment

None

### What browser(s) / client(s) have you tested

- [ ] 💥 Internet Explorer
- [X] 💥 Microsoft Edge
- [X] 💥 Google Chrome
- [ ] 💥 FireFox
- [ ] 💥 Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [ ] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Additional environment details

- browser version
- SPFx version
- Node.js version
- etc

### Describe the bug / error

We have observed that the PNP Powershell module used for interactive Sign in is failing now 
![PNpError](https://github.com/user-attachments/assets/f7ca165b-a8e3-4f57-9951-c9b8cb41fd77)

### Steps to reproduce

1. Connect-PNPonline -url -interactive
2. Key in USer Cred 
3. Complete MFA
4. Error : App doesnot found

### Expected behavior

This was working till 9th September 2024

**Comment by chandan805** [comment]
App is already registered in tenant with all delegate permissions

**Comment by a1mery** [comment]
It is due to changes in PnP Management Shell registration in Microsoft 365.
Please see this link: [PnP Management Shell EntraID app is deleted : what should I do ?](https://github.com/pnp/powershell/issues/4250)

**Comment by Ashlesha-MSFT** [comment]
@chandan805 
Yes, Connect-pnponline -url -interactive is changed, you can replace the “Interactive” switch with “UseWebLogin”.
Refer the document https://www.sharepointdiary.com/2021/08/fix-connect-pnponline-parameter-cannot-be-found-that-matches-parameter-name-interactive-error.html
![Image](https://github.com/user-attachments/assets/d9f672f4-ccfa-490b-9156-8c969b025ca6)

**Comment by Ashlesha-MSFT** [comment]
We are closing this issue for now. If the problem persists, feel free to reopen it or open a new one.
